src/service-metrics.ts

```diff
@@ -0,0 +1,91 @@
+/* Service metrics need to push to a collector rather than expose
```

Comment:
I wonder if this should live in the ceramic metrics package.

Reply:
Yeah it should live in its own package - which doesn't exist yet

Reply:
Yes if anything else ever needs this kind of metrics, we should make a repo just for these public metrics packages, I think - that way can quickly change them